src/main/webapp/app/service/firebase/firebase-gene-service.ts

```diff
@@ -410,17 +411,57 @@ export class FirebaseGeneService {
     });
   };
 
-  addGenomicIndicator = async (genomicIndicatorsPath: string) => {
+  addEmptyGenomicIndicator = async (genomicIndicatorsPath: string) => {
+    await this.addGenomicIndicator(true, genomicIndicatorsPath, '');
+  };
+
+  addGenomicIndicator = async (
+    skipReview: boolean,
+    genomicIndicatorsPath: string,
+    name: string,
+    description?: string,
+    alleleStates?: ALLELE_STATE[],
+  ) => {
+    const genePath = parseFirebaseGenePath(genomicIndicatorsPath);
     const newGenomicIndicator = new GenomicIndicator();
+    const uuidsToReview = [];
 
     newGenomicIndicator.associationVariants = [{ name: PATHOGENIC_VARIANTS, uuid: PATHOGENIC_VARIANTS }];
 
     const newReview = new Review(this.authStore.fullName);
     newReview.updateTime = new Date().getTime();
-    newReview.added = true;
-    newGenomicIndicator.name_review = newReview;
 
-    await this.firebaseRepository.pushToArray(genomicIndicatorsPath, [newGenomicIndicator]);
+    newGenomicIndicator.name = name;
```

Comment:
It looks like you do a bunch of work to create the review even if `skipReview === true`. The resources required for this work are definitely negligible, but can we only do it if `skipReview === false`?

Reply:
I changed the name to toReivew so I don't have to do a lot of `!skipReview`